### ISSUE TITLE
using sport_ecu combined with Naze32 INAV or Betaflight

### ISSUE THREAD

**Issue by Quaxpilot** [raphaelcoeffic/sport_ecu#1]
Hi ,
cool project!
I´m testing the flightcontroller as Naze32 or similar.
It would be great to implement your code into tha main Coding from INAV or Betaflight to build in a FLightcontroller and get via Naze32 the cu data as well , is that possible?
https://github.com/cleanflight/cleanflight
Thanks Thorsten

**Comment by raphaelcoeffic** [comment]
@Quaxpilot Are you writing about using a flight controller in a turbine jet? That sounds kind of funny! Is that about thrust vectoring or something like that?

**Comment by Quaxpilot** [comment]
Yes i have a turbine rc model it´s a conventional fixed turbine. On the other hand i´m testing using flightcontroller with inav first of all in epp models - very nice this stuff this fligcontrollers like autopilot etc.. I thing we could use a flightcontroller in rc jet´s too.  You made this sport ecu project maybee it´s possible to do the script into the code from inav oder betaflight for getting the data complete in flight. I my case i ordered a taranis x9e for checking telemetry ...
